magefiles/helm.go

```diff
@@ -100,9 +101,13 @@ func updateChartReleaseNotes(releasesNotes []string) {
 	Info("HELM Updating the Chart Release notes")
 	chart, err := chartutil.LoadChartfile(HelmChartPath)
 	CheckIfError(err, "HELM Could not Load Chart to update release notes %s", HelmChartPath)
-	var releaseNoteString string
+	releaseNoteString := ""
 	for i := range releasesNotes {
-		releaseNoteString = fmt.Sprintf("%s - %s\n", releaseNoteString, releasesNotes[i])
+		if len(releaseNoteString) == 0 {
+			releaseNoteString = fmt.Sprintf("- \"%s\"\n", releasesNotes[i])
+		} else {
+			releaseNoteString = fmt.Sprintf("%s- \"%s\"\n", releaseNoteString, releasesNotes[i])
+		}
```

Comment:
```suggestion
		releaseNoteString = fmt.Sprintf("%s- \"%s\"\n", releaseNoteString, releasesNotes[i])
```

No need to add additional judgment

Reply:
it add |2 if not, since the first releasenotestring is empty.

Reply:
The first iteration of the loop will always result in an empty releaseNoteString. 

I don't know why it will add `|2`